src/export/Export.h

```diff
@@ -263,9 +263,8 @@ class  AUDACITY_DLL_API Exporter final : public wxEvtHandler
    int mFilterIndex;
    int mFormat;
    int mSubFormat;
-   int mNumSelected;
-   unsigned mNumLeft;
-   unsigned mNumRight;
+   int mNumSelected{};
```

Comment:
So I suppose this will initialize `mNumSelected` to 0 and `mMono` to false? I prefer explicit assignments.